nixos/modules/virtualisation/hcloud-image.nix

```diff
@@ -0,0 +1,34 @@
+{
+  config,
+  lib,
+  pkgs,
+  ...
+}:
+
+let
+  cfg = config.hcloud;
+in
+
+{
+  imports = [
+    ./hcloud-config.nix
+    ../image/file-options.nix
+  ];
+
+  config.system.nixos.tags = [ "hcloud" ];
+  config.system.build.image = config.system.build.hcloudImage;
+  config.image.extension = "raw";
+
+  config.system.build.hcloudImage = import ../../lib/make-disk-image.nix {
```

Comment:
What's the benefit of adding yet another attribute here? I think config.system.build.image should be enough?

The reason other modules have non-generically named attributes aas well is that those were added before config.system.build.image was a thing.

Reply:
I didn't know! Fixed it. 🙂